### ISSUE TITLE
A11y_SQLAzureDataStudio_SQLServerBigDataCluster_ServiceSettings_AI4D:The control of the information icon is set as custom.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#9295]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11y_SQLAzureDataStudioBenchmark;#A11yMAS;#A11ySev2;#A11yTCS;#SQL Azure Data Studio;#Benchmark;#DesktopApp;#Win32;#MAS1.3.1;#AI4D;#MAS4.1.2;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.16.0-insider
OS: Windows_NT x64 10.0.18363

**Additional Details:**
Tool:Accessibility insights for desktop
MAS References:MAS4.1.2, MAS1.3.1

**Repro Steps:**
1. Launch Azure Data Studio.
2. Select the Connections icon (if not already selected) from left bar.
3. Click on the ellipses '...' and select "New Deployment".
4. Select "SQL Server Big Data Cluster" card under deployment options.
5. Select the version & checkbox.
6. Navigate till Selet button & hit enter.
7. Select Configuration profile --> Update Azure settings --> Update Cluster settings & hit Next.
8. Launch accessibility insights for desktop .

**Actual:**
Issue1:
The control type for information icon present under button "Storage settings" beside controller is set as "Custom" 

Issue2:
The name of the information icon is set as the details present in it.

**Expected:**
Issue1:
The control type should be set as a button .

Issue2:
The name should be unique and not the content of the tooltip.

**User Impact:**
Screen reader users will not know what actions can be taken on it since its control is not announced.

**Attachment for Reference :**
![image](https://user-images.githubusercontent.com/24580570/75137846-b66d5a80-570e-11ea-9d99-0c43f1ec7b2f.png)

**Comment by BhanuPrakashGit** [comment]
As "SQL Server Big Data Cluster" feature has been removed.
**GithubTags:**#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
